.eslintrc.js

```diff
@@ -1,11 +1,12 @@
 module.exports = {
-  extends: ["@stellar/eslint-config"],
+  extends: ["@stellar/eslint-config", "plugin:import/typescript"],
   rules: {
     "no-console": "off",
     "import/no-unresolved": "off",
     "react/jsx-filename-extension": ["error", { extensions: [".tsx", ".jsx"] }],
     "no-plusplus": "off",
     "jsdoc/check-indentation": "off",
+    camelcase: "off",
```

Comment:
just curious, why did this need to be disabled? Wondering if it's worth disabling it inline for whatever code is causing the error rather than turning it off altogether

Reply:
👍 Will remove this and will fix inline as needed.